experimental/telemetry/doc.go

```diff
@@ -35,13 +35,18 @@
 //     if err != nil {
 //     p.API.LogWarn("telemetry client not started", "error", err.Error())
 //     }
+//     ...
+//     p.tracker = telemetry.NewTracker(p.telemetryClient, p.API.GetDiagnosticId(), p.API.GetServerVersion(), manifest.Id,
+//     manifest.Version, "pluginName", enableDiagnostics, logger)
 //     }
 //  3. Init and update the tracker on configuration change
 //     func (p *Plugin) OnConfigurationChange() error {
 //     ...
-//     p.tracker = telemetry.NewTracker(p.telemetryClient, p.API.GetDiagnosticId(), p.API.GetServerVersion(), manifest.Id,
-//     manifest.Version, "pluginName", enableDiagnostics, logger)
-//     return nil
+//     if enableDiagnostics && p.tracker != nil {
+//     p.tracker.Enable()
+//     } else if !enableDiagnostics && p.tracker != nil {
+//     p.tracker.Disable()
+//     }
```

Comment:
1/5 it's easier to read if 
```suggestion
//  if p.tracker != nil {
//      if enableDiagnostics {
//          p.tracker.Enable()
//      } else {
//          p.tracker.Disable()
//      }
//  }
```